test.js

```diff
@@ -21,7 +19,7 @@ require('abstract-leveldown/abstract/get-test').all(MemDOWN, test, testCommon)
 
 require('abstract-leveldown/abstract/put-test').all(MemDOWN, test, testCommon)
```

Comment:
Could remove this `testCommon` parameter as well if you like.

Reply:
Ah yes :) will do it in this PR